src/main/kotlin/no/nav/familie/ef/sak/utestengelse/UtestengelseDto.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.familie.ef.sak.utestengelse
+
+import no.nav.familie.kontrakter.felles.Månedsperiode
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class UtestengelseDto(
+    val id: UUID,
+    val periode: Månedsperiode,
+    val slettet: Boolean,
+    val opprettetAv: String,
+    val opprettetTid: LocalDateTime,
+    val endretAv: String,
+    val endretTid: LocalDateTime,
```

Comment:
Er det mulig å endre en utestengelse? Hvis ikke kan disse to fjernes fra dto-en

Reply:
Hvis man fjerner utestengelsen så endres endretAv/endretTid

Reply:
Men skal det brukes til noe? PR-en på frontend bruker foreløpig bare periode og id?

Reply:
Tenker det kan være fint med historikk hvis det noen gang blir diskusjon om hva som har skjedd.

Reply:
Så lenge som man kan fjerne (dvs endre) så tenker jeg vi må ha det. Nå har vi hatt litt diskusjon om at vi kanskje skal gå for å ha utestengelseperioder i en "behandlingsflyt", då blir det kanskje ikke interessant